src/flag_gems/csrc/cstub.cpp

```diff
@@ -70,6 +70,18 @@ TORCH_LIBRARY(flag_gems, m) {
       "reshape_and_cache_flash(Tensor key, Tensor value, Tensor(a!) key_cache, Tensor(b!) value_cache, "
       "Tensor slot_mapping, str kv_cache_dtype, Tensor? k_scale=None, Tensor? v_scale=None) -> "
       "()");
+  m.def(
+      "flash_attn_varlen_func(Tensor q, Tensor k, Tensor v, SymInt max_seqlen_q, Tensor cu_seqlens_q, SymInt "
+      "max_seqlen_k, "
+      "Tensor? cu_seqlens_k=None, Tensor? seqused_k=None, Tensor? q_v=None, float dropout_p=0.0, float? "
+      "softmax_scale=None, "
+      "bool causal=False, SymInt[]? window_size=None,float softcap=0.0, "
+      "Tensor? alibi_slopes=None, "
+      "bool deterministic=False, bool return_attn_probs=False, Tensor? block_table=None, bool "
+      "return_softmax_lse=False, "
+      "Tensor? out=None, Tensor? scheduler_metadata=None, float? q_descale=None, float? k_descale=None, "
+      "float? v_descale=None, "
+      "SymInt fa_version=2) -> (Tensor, Tensor)");
```

Comment:
The signature does not correspond to the function signature in flash_attn_varlen_func.cpp.